mod/metrics/set/abstract/answer_search.rb

```diff
@@ -44,7 +44,7 @@ def item_view_from_query
     merge_filter_defaults
     wrap_with :div, class: "filter-form-and-result nodblclick" do
       class_up "card-slot", "_filter-result-slot"
-      output [_render_filter_form, _render_filter_result]
+      output [_render_filter_form, _render_filter_result(home_view: :filter_result)]
```

Comment:
This is a hack to get around what I see as a decko bug in the handling of home views.  The filter result slot doesn't get :filter_result as its home view by default.